core/test/ragtime/core_test.clj

```diff
@@ -23,6 +23,14 @@
     (run-up! [_ store] (swap! (:data store) assoc key val))
     (run-down! [_ store] (swap! (:data store) dissoc key))))
 
+(defn interrupted-migration []
+  (reify p/Migration
+    (id [_] "interrupted-migration")
+    (run-up! [_ store]
+      (.interrupt (Thread/currentThread))
+      (swap! (:data store) assoc :interrupted? true))
+    (run-down! [_ store] (swap! (:data store) dissoc :interrupted?))))
```

Comment:
This would be more consistent with the previous method if it were written:

```clojure
(run-down! [_ stpre]
  (swap! (:data store) dissoc :interrupted?))
```